src/main/java/pages/cartPage/CartPage.java

```diff
@@ -15,9 +15,10 @@ public CartPage() {
         this.basePage = new BasePage();
     }
 
-    ApplyCouponCodeTable applyCouponCodeTable = new ApplyCouponCodeTable();
-    EstimateShippingAndTaxesTable estimateShippingAndTaxesTable = new EstimateShippingAndTaxesTable();
-    TotalsTable totalsTable = new TotalsTable();
+    public ApplyCouponCodeTable applyCouponCodeTable = new ApplyCouponCodeTable();
+    public EstimateShippingAndTaxesTable estimateShippingAndTaxesTable = new EstimateShippingAndTaxesTable();
+    public TotalsTable totalsTable = new TotalsTable();
+    public ShoppingCartTable shoppingCartTable = new ShoppingCartTable();
```

Comment:
@Fishgame1 just curious, you mentioned using vars before - should vars be used in such cases as well?

Reply:
@jakub-krzysiek No, you cannot use ```var``` to declare class fields The ```var```can only be used to declare local variables with initialisation.